privacysandbox/topics.go

```diff
@@ -0,0 +1,205 @@
+package privacysandbox
+
+import (
+	"encoding/json"
+	"strconv"
+	"strings"
+
+	"github.com/prebid/openrtb/v20/openrtb2"
+	"github.com/prebid/prebid-server/v2/util/jsonutil"
+)
+
+type Topic struct {
+	SegTax   int    `json:"segtax,omitempty"`
+	SegClass string `json:"segclass,omitempty"`
+	SegIDs   []int  `json:"segids,omitempty"`
+}
+
+// ParseTopicsFromHeader parses the Sec-Browsing-Topics header data into Topics object
+func ParseTopicsFromHeader(secBrowsingTopics string) []Topic {
+	var topics []Topic
```

Comment:
Would it be worth it to pre-allocate this slice to size 10, if that's the number you break at within the for loop line 27?